lxd/db/db.go

```diff
@@ -164,16 +164,27 @@ func OpenCluster(name string, dialer grpcsql.Dialer, address string) (*Cluster,
 	// Test that the cluster database is operational. We wait up to 10
 	// minutes, in case there's no quorum of nodes online yet.
 	timeout := time.After(10 * time.Minute)
-	for {
+	for i := 0; ; i++ {
+		// Log initial attempts at debug level, but use warn
+		// level after the 10'th attempt (about 10 seconds).
+		log := logger.Debugf
```

Comment:
Same comment as the other one, please don't alias the logger.* functions as I often grep for them

Reply:
Fixed with the same pattern as above.